layout/theme.liquid

```diff
@@ -108,6 +108,10 @@
       <link rel="preload" as="font" href="{{ settings.type_header_font | font_url }}" type="font/woff2" crossorigin>
     {%- endunless -%}
 
+    {%- if settings.predictive_search_enabled -%}
+      <link rel="preload" href="{{ 'component-predictive-search.css' | asset_url }}" as="style" onload="this.onload=null;this.rel='stylesheet'">
```

Comment:
```suggestion
      <link rel="preload" href="{{ 'component-predictive-search.css' | asset_url }}" as="style" onload="this.onload=null;this.rel='stylesheet'">
```

Here we don't want to reuse the same approach as we use for other stylesheets ? 
`<link rel="stylesheet" href="{{ 'component-menu-drawer.css' | asset_url }}" media="print" onload="this.media='all'">`